packages/smooth_app/lib/pages/preferences/user_preferences_settings.dart

```diff
@@ -54,9 +59,115 @@ class UserPreferencesSettings extends AbstractUserPreferences {
         _ProductsSettings(),
         _MiscellaneousSettings(),
         _PrivacySettings(),
+        _RateUs(),
+        _ShareWithFriends(),
       ];
 }
 
+class _RateUs extends StatelessWidget {
```

Comment:
is it conditional to the flavor ? We also build for fdroid, fyi

Reply:
I will check that ,

Reply:
flutter doesn't give platform detection for f-droid
![image](https://user-images.githubusercontent.com/93595710/218765496-9cca0755-bfdf-4fe4-bcf3-7ecb8383bea4.png)

Reply:
Can you explain a bit more about "is it conditional to the flavor"?

Reply:
If this also works with F-Droid and I think it does not


You can get the flavour using the global variable `flavour` (main.dart l.54) can be used everywhere in the code.


Or even better `await ApplicationStore.openAppReview()` we already have this function implemented, this way you don't even have to worry about this and we don't need store_redirect

Reply:
Oh coooll I will look into it and make changes !